spec/factories/book_suggestion.rb

```diff
@@ -0,0 +1,12 @@
+FactoryBot.define do
+  factory :book_suggestion do
+    synopsis { Faker::Lorem.paragraph }
+    price { Faker::Commerce.price(as_string: true) }
+    author { Faker::Book.author }
+    title { Faker::Book.title }
+    link { Faker::Internet.url }
+    editor { Faker::Book.publisher }
+    year { Faker::Number.within(range: 0..2019) }
+    user { build(:user) }
```

Comment:
FactoryBot can infer nested model, and use user factory. In this case, this sentence
```rb
    user { build(:user) }
```
It's the same as this another:
```rb
    user
```

```suggestion
    user
```